### DISCUSSION TITLE
Default editor per tag

### DISCUSSION THREAD

**Discussion by ghost** [standardnotes/forum#2647] [Suggestions] [answered]
I almost always use the same editor per tag (e.g. Code Snippets -> Code Editor, Todo -> Task Editor, etc). It would be great to set a default editor per tag instead of a default editor for all notes.

**Comment by JaspalSuri** [comment]
Thanks for the feedback, @commitmentbasedsmartcontract. We might consider this after tags (i.e. folders) become a native feature. 🙂

**Comment by ghost** [comment]
Great, thanks

**Comment by myreli** [comment, marked as answer]
This is now available:

<img width="674" alt="image" src="https://user-images.githubusercontent.com/17554234/218860113-cc4e9e8c-8a25-4a7b-b7b5-f2b83369fcab.png">
